openstack/loadbalancer/v2/monitors/requests.go

```diff
@@ -108,16 +108,14 @@ type CreateOpts struct {
 	MaxRetries int `json:"max_retries" required:"true"`
 
 	// URI path that will be accessed if Monitor type is HTTP or HTTPS.
-	// Required for HTTP(S) types.
 	URLPath string `json:"url_path,omitempty"`
 
 	// The HTTP method used for requests by the Monitor. If this attribute
 	// is not specified, it defaults to "GET". Required for HTTP(S) types.
 	HTTPMethod string `json:"http_method,omitempty"`
 
 	// Expected HTTP codes for a passing HTTP(S) Monitor. You can either specify
-	// a single status like "200", or a range like "200-202". Required for HTTP(S)
-	// types.
+	// a single status like "200", or a range like "200-202".
```

Comment:
Well, just for the sake of correctness - such strings work with Octavia as well:

"200, 202"
"200-202, 401"

Reply:
Thanks! This has been updated.